libr/core/cmd_debug.c

```diff
@@ -771,6 +771,20 @@ static int step_until_esil(RCore *core, const char *esilstr) {
 	return true;
 }
 
+static bool is_repeatalbe_inst(RDebug *dbg) {
+	RAnalOp op;
+	ut64  pc;
+	ut8 buf[32];
+	pc = r_debug_reg_get (dbg, dbg->reg->name[R_REG_NAME_PC]);
```

Comment:
Theres a function in core that gives u a ranalop by giving the address. Nonneed to do the ioread+anal by hand every time

Reply:
done